src/features/Map/ui/Map.tsx

```diff
@@ -1,28 +1,74 @@
-import React, { useEffect, useRef } from "react";
-import { YaMap, Geocoder } from "react-native-yamap";
+import React, { useEffect, useRef, useState } from "react";
+import { YaMap } from "react-native-yamap";
 import { styles } from "./Map.styles";
 import { useSetlocationStore } from "@entities/user";
-import { useSetCurretLocationStore } from "@entities/user";
-import { Nullable } from "@shared/api";
+import { Nullable, ParkingInf, ParkingSchema } from "@shared/api";
+import { ParkingMarker } from "@features/ParkingMarker";
+import { ParkingInfModal } from "@shared/ui";
 
-export const Map = () => {
-    const { currentLocation } = useSetCurretLocationStore();
+interface Map {
```

Comment:
leave Map as a separate component only about map and add modal in screen file. Make map reusable